lsp-headerline.el

```diff
@@ -92,12 +92,12 @@ for caching purposes.")
   (let ((filename (f-filename file-path))
         (file-ext (f-ext file-path)))
     (if file-ext
-        (when (require 'lsp-treemacs nil t)
-          (if-let (icon (lsp-treemacs-get-icon file-ext))
-              (format "%s %s"
-                      (lsp-headerline--fix-image-background icon)
-                      filename)
-            filename))
+        (if (require 'lsp-treemacs nil t)
+            (if-let (icon (lsp-treemacs-get-icon file-ext))
```

Comment:
The `(if-let` needs a matching `else` clause to avoid having the function return nil.

Reply:
done